airbyte-integrations/connectors/source-file/pyproject.toml

```diff
@@ -3,7 +3,7 @@ requires = [ "poetry-core>=1.0.0",]
 build-backend = "poetry.core.masonry.api"
 
 [tool.poetry]
-version = "0.5.2"
+version = "0.5.3"
```

Comment:
Can you run poetry lock in the source-file folder to update the poetry.lock file?

Reply:
Updated poetry.lock.
About your previous question: when reading a zip file, the encoding parameter will still be used - but only inside the load_dataframes method, when the file is already decompressed. It will be passed to the pandas reader along with other arguments inside the reader_options dictionary.